diff --git a/meta-hailo-accelerator/conf/layer.conf b/meta-hailo-accelerator/conf/layer.conf
index a74232f..050cf90 100644
--- a/meta-hailo-accelerator/conf/layer.conf
+++ b/meta-hailo-accelerator/conf/layer.conf
@@ -10,4 +10,4 @@ BBFILE_PATTERN_meta-hailo-accelerator = "^${LAYERDIR}/"
 BBFILE_PRIORITY_meta-hailo-accelerator = "8"
 
 LAYERDEPENDS_meta-hailo-accelerator = "core"
-LAYERSERIES_COMPAT_meta-hailo-accelerator = "kirkstone"
+LAYERSERIES_COMPAT_meta-hailo-accelerator = "kirkstone langdale"
diff --git a/meta-hailo-libhailort/conf/layer.conf b/meta-hailo-libhailort/conf/layer.conf
index e9e0ceb..801ad18 100644
--- a/meta-hailo-libhailort/conf/layer.conf
+++ b/meta-hailo-libhailort/conf/layer.conf
@@ -10,4 +10,4 @@ BBFILE_PATTERN_meta-hailo-libhailort = "^${LAYERDIR}/"
 BBFILE_PRIORITY_meta-hailo-libhailort = "8"
 
 LAYERDEPENDS_meta-hailo-libhailort = "core"
-LAYERSERIES_COMPAT_meta-hailo-libhailort = "kirkstone"
+LAYERSERIES_COMPAT_meta-hailo-libhailort = "kirkstone langdale"
diff --git a/meta-hailo-libhailort/recipes-hailo/libhailort/libhailort_4.19.0.bb b/meta-hailo-libhailort/recipes-hailo/libhailort/libhailort_4.19.0.bb
index f7bac01..0c2f65a 100644
--- a/meta-hailo-libhailort/recipes-hailo/libhailort/libhailort_4.19.0.bb
+++ b/meta-hailo-libhailort/recipes-hailo/libhailort/libhailort_4.19.0.bb
@@ -25,7 +25,7 @@ do_install:append() {
 
   install -d ${HAILORT_EXPORT_DIR}
   install -m 0644 ${WORKDIR}/build/hailort/libhailort/src/*.cmake ${HAILORT_EXPORT_DIR}
-  install -m 0644 ${WORKDIR}/build/hailort/libhailort/src/CMakeFiles/Export/lib/cmake/HailoRT/*.cmake ${HAILORT_EXPORT_DIR}
+  install -m 0644 ${WORKDIR}/build/hailort/libhailort/src/CMakeFiles/Export/*/*.cmake ${HAILORT_EXPORT_DIR}
 
 }
 
diff --git a/meta-hailo-tappas/conf/layer.conf b/meta-hailo-tappas/conf/layer.conf
index 8e8f46b..602d801 100644
--- a/meta-hailo-tappas/conf/layer.conf
+++ b/meta-hailo-tappas/conf/layer.conf
@@ -12,6 +12,6 @@ BBFILE_PRIORITY_meta-hailo-tappas = "8"
 LICENSE_PATH = "${LAYERDIR}/licenses/"
 
 LAYERDEPENDS_meta-hailo-tappas = "core meta-hailo-libhailort"
-LAYERSERIES_COMPAT_meta-hailo-tappas = "kirkstone"
+LAYERSERIES_COMPAT_meta-hailo-tappas = "kirkstone langdale"
 
 CORE_IMAGE_EXTRA_INSTALL:append = " packagegroup-hailo-tappas${@bb.utils.contains('HAILO_TARGET', 'core-image-hailo', '', '-dev-pkg', d)}"
diff --git a/meta-hailo-tappas/recipes-gstreamer/tappas-apps/tappas-apps_3.30.0.bb b/meta-hailo-tappas/recipes-gstreamer/tappas-apps/tappas-apps_3.30.0.bb
index 964e1d5..ac054a4 100644
--- a/meta-hailo-tappas/recipes-gstreamer/tappas-apps/tappas-apps_3.30.0.bb
+++ b/meta-hailo-tappas/recipes-gstreamer/tappas-apps/tappas-apps_3.30.0.bb
@@ -28,25 +28,25 @@ ROOTFS_APPS_DIR = "${D}/home/root/apps"
 
 APPS_DIR_PREFIX = "${WORKDIR}/git/apps/"
 IMX8_DIR = "${APPS_DIR_PREFIX}/h8/gstreamer/imx8/"
-HAILO15_DIR = "${APPS_DIR_PREFIX}/h15/gstreamer/"
+#HAILO15_DIR = "${APPS_DIR_PREFIX}/h15/gstreamer/"
 
 REQS_PATH = "${FILE_DIRNAME}/files/"
 REQS_IMX8_FILE = "${REQS_PATH}download_reqs_imx8.txt"
-REQS_HAILO15_FILE = "${REQS_PATH}download_reqs_hailo15.txt"
+#REQS_HAILO15_FILE = "${REQS_PATH}download_reqs_hailo15.txt"
 
 REQS_FILE = ""
 ARM_APPS_DIR = ""
 python () {
-    if 'imx8' in d.getVar('MACHINE'):
+    #if 'imx8' in d.getVar('MACHINE'):
         d.setVar('REQS_FILE', d.getVar('REQS_IMX8_FILE'))
         d.setVar('ARM_APPS_DIR', d.getVar('IMX8_DIR'))
-    else:
-        d.setVar('REQS_FILE', d.getVar('REQS_HAILO15_FILE'))
-        d.setVar('ARM_APPS_DIR', d.getVar('HAILO15_DIR'))
-        d.appendVar('DEPENDS', " libmedialib-api xtensor")
+    #else:
+    #    d.setVar('REQS_FILE', d.getVar('REQS_HAILO15_FILE'))
+    #    d.setVar('ARM_APPS_DIR', d.getVar('HAILO15_DIR'))
+    #    d.appendVar('DEPENDS', " libmedialib-api xtensor")
 }
 
-IS_H15 = "${@ 'true' if 'hailo15' in d.getVar('MACHINE') else 'false'}"
+#IS_H15 = "${@ 'true' if 'hailo15' in d.getVar('MACHINE') else 'false'}"
 INSTALL_LPR = "true"
 
 CURRENT_APP_NAME = ""
